File: Coding/03_Beginner/Full_search/Ruby/15.rb
N, M = gets.split.map(&:to_i)
A = gets.split.map(&:to_i)
B = gets.split.map(&:to_i)

cnt = 0
A.each do |a|
    B.each do |b|
        if a > b
            cnt += 1
        end
    end
end

puts cnt

File: Coding/03_Beginner/Full_search/C++/15.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
    int N, M;
    cin >> N >> M;
    vector<int> A(N), B(M);

    for (int i = 0; i < N; i++) cin >> A[i];
    for (int j = 0; j < M; j++) cin >> B[j];

    int cnt = 0;
    for (int i = 0; i < N; i++){
        for (int j = 0; j < M; j++){
            if (A[i] > B[j]) cnt++;
        }
    }

    cout << cnt << endl;
    return 0;
}